lib/features/children/parental_approval/widgets/parental_approval_declined_page.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:givt_app/features/children/family_history/models/child_donation.dart';
+import 'package:givt_app/l10n/l10n.dart';
+import 'package:givt_app/utils/app_theme.dart';
+
+class ParentalApprovalDeclinedPage extends StatelessWidget {
+  const ParentalApprovalDeclinedPage({
+    required this.donation,
+    super.key,
+  });
+
+  final ChildDonation donation;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      mainAxisSize: MainAxisSize.min,
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        Text(
+          '${context.l10n.childParentalApprovalDeclinedTitlePrefix}${donation.name}${context.l10n.childParentalApprovalDeclinedTitlePostfix}',
```

Comment:
If you want to compose strings like this it's better to do it in PO editor. By adding "My string {value0} with second {value1}" and in code you just call "context.l10n" and then you get the option to add 2 parameters and in this way if the text changes you don't have to make code changes.

Reply:
[Example](https://github.com/givtnl/givt-app/blob/528c626282592cf10b82f5e038967f41b41e27d0/lib/l10n/arb/app_en_US.arb#L130C18-L130C18)

Like this one for example

Reply:
super nice, didn't know about that, thank you!

Reply:
addressed. once again thanks, now it looks much nicer :)